kinto_attachment/tests/__init__.py

```diff
@@ -62,14 +62,22 @@ def make_app(self):
         return app
 
     def upload(self, files=None, params=[], headers={}, status=None,
-               randomize=True):
+               randomize=True, gzipped=False):
         files = files or self.default_files
         headers = headers or self.headers.copy()
         content_type, body = self.app.encode_multipart(params, files)
         headers['Content-Type'] = core_utils.encode_header(content_type)
 
+        params = {}
         if not randomize:
-            endpoint_url = self.endpoint_uri + '?randomize=false'
+            params['randomize'] = 'false'
+
+        if gzipped:
+            params['gzipped'] = 'true'
+
+        if len(params) > 0:
+            params = ['%s=%s' % (k, v) for k, v in params.items()]
```

Comment:
We probably want to use [urllib.urlencode](http://docs.python.org/library/urllib.html#urllib.urlencode) here.